preferences/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+
+from django.utils.translation import gettext_lazy as _
+from django.conf import settings
+
+# for user preferences
+class Preferences (models.Model):
```

Comment:
I believe adding a ‘user = models.OneToOne(…)’ field on the ‘Preferences’ model will make the view.py code simpler.